tests/deprecated_api/test_remove_1-5_metrics.py

```diff
@@ -20,7 +20,7 @@
 from pytorch_lightning.metrics.utils import get_num_classes, select_topk, to_categorical, to_onehot
 
 
-def test_v1_5_0_metrics_utils():
+def test_v1_5_metrics_utils():
     x = torch.tensor([1, 2, 3])
     with pytest.deprecated_call(match="It will be removed in v1.5.0"):
```

Comment:
I would expect this message to tell where is now placed the new function and how do I call it

Reply:
yes it is there, which is just a fraction of the message :]

Reply:
awesome :)